src/plugins/vis_type_vega/public/data_model/utils.test.js

```diff
@@ -91,4 +91,36 @@ describe('Utils.handleInvalidQuery', () => {
       }).toThrowError();
     });
   });
+
+  test('should identify object contains function properties', async () => {
+    const input1 = {
+      key1: 'value1',
+      key2: () => {
+        alert('Hello!');
+      },
+    };
+
+    expect(Utils.checkForFunctionProperty(input1)).toBe(true);
+
+    const input2 = {
+      key2: () => {
+        alert('Hello!');
+      },
```

Comment:
could we use other method other than alert() in unit test. looks weird UT code has test dependencies on UI function.

just we just be?
```suggestion
      key2: () => {},
```

Reply:
Replaced with jest.fn() which is much better. I was copying the same pattern as other tests in this fixture.